projects/kit/components/select/select.component.ts

```diff
@@ -54,7 +53,12 @@ export class TuiSelectComponent<T>
 
     @Input()
     @tuiDefaultProp()
-    identityMatcher: TuiIdentityMatcher<T | string> = TUI_DEFAULT_IDENTITY_MATCHER;
+    stringify: TuiSelectOptions<T>['stringify'] = this.options.stringify;
```

Comment:
Technically `stringify` is unnecessary due to `valueContent` but it is probably cool to add for uniformity. What do you think, @splincode @vladimirpotekhin ?